src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -5335,10 +5399,12 @@ public override object PayloadValue(int index)
             switch (index)
             {
                 case 0:
-                    return TypeID;
+                    return TypeName;
```

Comment:
❓ What happens when this code tries to load data produced by an earlier version of the tool?